### DISCUSSION TITLE
All my saved works could not be found

### DISCUSSION THREAD

**Discussion by Shicoren** [jgraph/drawio#2602] [General]
Hi, 

I'm working with this app already for several months continuously on a daily basis - which means creating, modifying, saving, and again the loop 
today for the first time, I had to restart my laptop so I saved my work and closed the tab - before the restart I wanted to show my work to a colleague and suddenly I see this message 
![image](https://user-images.githubusercontent.com/98691401/151697845-ea2fca02-94ee-4887-97c2-8e8aa2c3eed2.png)

I tried all the ways to restore my work and didn't get succeed - even if opening the recent files or the closed tab, nothing helped me 
please help me as I had so many works there 
thanks

**Comment by davidjgraph** [comment]
The most likely issue, as the dialog suggests, is that the file does not eixst or you do not have access to it.

Go to drive.google.com logged in as that user and check you can find the file there. Did you create the file? Did you ever give anyone else write permissions? Is this a company Google Workspace?

**Reply by Shicoren** [reply]
Hi David, 
thank you for your reply.
I tried to be logged in (as always I am, but this time I even didn't close the tab of google drive and still cannot find it) 
I've created the file and didn't give anyone else permission... it was like 3-4 months of work with always saving it - 
the google workspace is for a company but exactly like personal usage - we are a small company 
Is there any chance you have some copy of it in a cache or somewhere else? 

thanks

**Reply by davidjgraph** [reply]
We don't have access to your diagram. If the diagram is not available under drive.google.com, then it's been deleted or Google has lost it.
